socorro/processor/rules/mozilla.py

```diff
@@ -268,24 +268,49 @@ class JavaProcessRule(Rule):
     """Move Java-crash-related bits over."""
 
     def predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        return bool(raw_crash.get("JavaStackTrace", None))
+        return bool(raw_crash.get("JavaStackTrace", None)) or bool(
+            raw_crash.get("JavaException", None)
+        )
```

Comment:
Part of me wonders whether we should move all the Java crash related rules to a new file. I'm deferring doing that until the next Java-specific change we make.